.github/workflows/release.yml

```diff
@@ -31,31 +33,47 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: "${{ needs.go-version.outputs.version }}"
+      - name: Generate Release Notes
+        # Fetch CHANGELOG.md contents up to Git tag prior to this release, skipping top two lines
+        run: sed -n -e "1{/# /d;}" -e "2{/^$/d;}" -e "/# $(git describe --abbrev=0 --exclude="$(git describe --abbrev=0 --match='v*.*.*' --tags)" --match='v*.*.*' --tags | tr -d v)/q;p" CHANGELOG.md > ${{ env.RELEASE_NOTES }}
       - name: Install kubebuilder
         run: |
           curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${{ env.KUBEBUILDER_VERSION }}/kubebuilder_${{ env.KUBEBUILDER_VERSION }}_linux_amd64.tar.gz -o /tmp/kb.tgz
           tar zxf /tmp/kb.tgz -C /tmp/
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v3.0.0
+        uses: goreleaser/goreleaser-action@v3
```

Comment:
It's good practice to pin versions down in "production" envs. If we're going to allow rolling versions for dependencies, I would at least pin down major and minor and only allow rolling on patch level.

This is to avoid subtle differences between 3.x and 3.x+1 causing us issues that are usually difficult to unconver.

Reply:
Sure, I will revert it. Thanks!